Makefile

```diff
@@ -4,30 +4,33 @@
 # Example:
 #   make build
 
-
-DIRS=$(shell find . -name project.json -exec dirname {} \;)
 OS=$(shell uname -s)
-LINUX_FRAMEWORK=netcoreapp1.0
-DEFAULT_FRAMEWORK=$(LINUX_FRAMEWORK)
 
+EXAMPLE_DIRS=$(shell find ./example -name *.csproj -exec dirname {} \;)
```

Comment:
This comment actually alerted me to a quite important mistake - should be finding in `./examples` not `./example`. Also, fixing the quotes.